website/docs/r/local_file_source.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_local_file_source"
+description: |-
+  Provides a Sumologic Local File Source.
+---
+
+# sumologic_local_file_source
+Provides a [Sumologic Local File Source][1].
+
+## Example Usage
+```hcl
+resource "sumologic_installed_collector" "installed_collector" {
+  name        = "test-collector"
+  category = "macos/test"
+  ephemeral = true
+}
+
+resource "sumologic_local_file_source" "local" {
+	name = "localfile-mac"
+	description = "test"
+	category = "test"
+	collector_id = "${sumologic_installed_collector.installed_collector.id}"
+  path_expression = "/Applications/Sumo Logic Collector/logs/*.log.*"
+}
+}
```

Comment:
indentation problem?

Reply:
still wrong indentation

Reply:
please be consistent with using tab or two spaces. in the previous resource you are using two spaces and in this resource it's a mix of tab and two spaces

Reply:
<img width="796" alt="image" src="https://user-images.githubusercontent.com/56007827/222854289-b92654fb-a792-4e54-966f-528cf1d2c7d5.png">